client/grpc/client.go

```diff
@@ -0,0 +1,50 @@
+package grpc
+
+import (
+	"errors"
+
+	"github.com/kava-labs/kava/client/grpc/query"
+	"github.com/kava-labs/kava/client/grpc/util"
+)
+
+// KavaGrpcClient enables the usage of kava grpc query clients and query utils
+type KavaGrpcClient struct {
+	config KavaGrpcClientConfig
+
+	// Query clients for cosmos and kava modules
+	Query *query.QueryClient
+
+	// Utils for common queries (ie fetch an unpacked BaseAccount)
+	*util.Util
```

Comment:
suggest "sdk" as a more intention revealing name here as we're builidng a higher level interface for making common dApp workflows simpler without having to make n number of calls to multiple api's to stitch together the end result that a dApp developer or kava engineer writing tests would care about